all.json

```diff
@@ -15,6 +15,44 @@
 		"1inch-wallet.org",
 		"1inch.tk",
 		"1metamask.com",
+		"accountvalidation.pages.dev",
+		"azukis.art",
+		"bayc-land.mints-nft.live",
+		"bnbfaucet.site",
+		"bonowatch.com",
+		"coinfactory.digital",
+		"comicsapesocialsclub.com",
+		"corriectionbot.co",
+		"cryptolightinvestment.com",
+		"eth-airdrops.io",
+		"faucetlists.ga",
+		"finras.info",
+		"foodmoodnft.io",
+		"laundapps-connect.us",
+		"linksafe-secure.com",
+		"livemints.events",
+		"mainnetconnect-wallets.com",
+		"metamask-cn.win",
+		"metamaskregister.com",
+		"metamaskwebs.net",
+		"metamaskwebs.phantomsapp.io",
+		"mindblowon.livemints.events",
```

Comment:
Not just a normal NFT? (the above main site falls into the same category)

Reply:
Fake NFT mint.
Any interaction with it result in devastating consequences for whoever was naive enough to let their web wallet interact with it.
Many also have fake Connect wallet seed stealing sub pages.
Gonna remov em .